DuggaSys/microservices/shared_microservices/getUid_ms.php

```diff
@@ -0,0 +1,28 @@
+<?php
+    date_default_timezone_set("Europe/Stockholm");
+    
+    // Include basic application services
+    include_once ("../../../../coursesyspw.php");
+    include_once ("../../../Shared/sessions.php");
+    include_once ("../../../Shared/basic.php");
+
+    function getUid(){
+        // Checks user id, if user has none a guest id is set
+        if(isset($_SESSION['uid'])){
+            $userid=$_SESSION['uid'];
+        }else{
+            $userid="1";
```

Comment:
Just a mention, but do we **always** want to default to `uid = 1`? Not completely sure what the different values represent. If `uid = 1` represent a guest then this should be fine, as long as it is consistent in all other services.